src/main/java/edu/unc/lib/boxc/migration/cdm/CdmIndexCommand.java

```diff
@@ -34,17 +41,31 @@ public class CdmIndexCommand implements Callable<Integer> {
 
     private CdmFieldService fieldService;
     private CdmIndexService indexService;
+    private ExportObjectsService exportObjectsService;
     private MigrationProject project;
 
+    @Mixin
+    private CdmIndexOptions options;
+
     @Override
     public Integer call() throws Exception {
         long start = System.nanoTime();
 
         try {
             initialize();
 
-            indexService.createDatabase(force);
-            indexService.indexAll();
+            // if user provides csv, check that it exists
+            if (options.getCsvFile() != null) {
+                if (Files.exists(options.getCsvFile())) {
+                    CdmFieldInfo csvExportFields = fieldService.retrieveFieldsFromCsv(options.getCsvFile());
+                    fieldService.persistCsvFieldsToProject(project, csvExportFields);
+                } else {
+                    throw new MigrationException("No csv file exists in " + options.getCsvFile());
+                }
+            }
+
+            indexService.createDatabase(force, options);
```

Comment:
just noticed we still have the `force` flag as a separate option, but we have CdmIndexOptions now. We should move it from this class into the options, and then createDatabase only needs one param